### PR TITLE
Add error message if eigen is not found at configure time (autotools).

### PR DESCRIPTION
**Pull request by fghoussen**
## Pull request purpose

fixing issue #183 ?

### CODE REVIEW

File: configure.ac [hunk 1]
```diff
@@ -77,7 +77,7 @@ AC_ARG_ENABLE([icb-exmm],
               [AS_VAR_SET([enable_icb_exmm], [no])])
 
 if test x"$enable_icb_exmm" != x"no"; then
-	PKG_CHECK_MODULES([EIGEN3], [eigen3 >= 3.2])
+	PKG_CHECK_MODULES([EIGEN3], [eigen3 >= 3.2], [], [AC_MSG_ERROR([Eigen not found. Give hints with PKG_CONFIG_PATH])])
```
Comment (sylvestre) [inline configure.ac]:
```suggestion
	PKG_CHECK_MODULES([EIGEN3], [eigen3 >= 3.2], [], [AC_MSG_ERROR([Eigen not found. Give hints with PKG_CONFIG_PATH. Disable the need of this dependency using --disable-icb-exmm])])
```

Comment (sylvestre) [reply configure.ac]: what do you think?

Comment (fghoussen) [reply configure.ac]: sure !

**Comment by fghoussen** [comment]
a change in configure.ac changes coverage ?... really ?...

**Comment by fghoussen** [comment]
Error added in configure (by configure.ac) is NOT covered (would break CI) : is this where the reg comes from ?!... :D

**Pull request merged**